selectors/rhs.jsx

```diff
@@ -41,3 +43,43 @@ export const getSelectedPost = createSelector(
         };
     }
 );
+
+export function makeGetCurrentUsersLatestPost(channelId, rootId) {
+    return createSelector(
+        getCurrentUserId,
+        (state) => state.entities.posts.postsInChannel[channelId] || [],
```

Comment:
I think it would be better to just return `state.entities.posts.postsInChannel[channelId]` and then check if `postIds` is empty below. Right now, if that part of the state is null, it'll return a new empty array each time this is called, preventing any memoization. Not a likely case, but I think it would be better to be safe here.

Reply:
👍